# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/webrtc/signaling/src/peerconnection/PeerConnectionCtx.h
# Commit: d4004105a04a
# Full Hash: d4004105a04ad9a473930acb9c6a05d90cd9f028
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-23 04:13:22
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r baecf588657c -r d4004105a04a media/webrtc/signaling/src/peerconnection/PeerConnectionCtx.h
--- a/media/webrtc/signaling/src/peerconnection/PeerConnectionCtx.h	Thu Feb 21 23:16:40 2019 +0000
+++ b/media/webrtc/signaling/src/peerconnection/PeerConnectionCtx.h	Thu Feb 21 20:24:54 2019 +0000
@@ -72,7 +72,9 @@
   std::map<const std::string, PeerConnectionImpl*> mPeerConnections;
 
   PeerConnectionCtx()
-      : mGMPReady(false), mTransportHandler(MediaTransportHandler::Create()) {}
+      : mGMPReady(false),
+        mTransportHandler(
+            MediaTransportHandler::Create(GetMainThreadSerialEventTarget())) {}
   // This is a singleton, so don't copy construct it, etc.
   PeerConnectionCtx(const PeerConnectionCtx& other) = delete;
   void operator=(const PeerConnectionCtx& other) = delete;